app/services/PageBuilder.scala

```diff
@@ -129,7 +129,7 @@ class PageBuilder extends ProcessPopulation {
   def fromPageDetails[A](pages: Seq[Page])(f: (String, String, String) => A): List[A] =
```

Comment:
If this method is only used by PageBuilderSpec I would move it into the test class.

Reply:
This is used in the backend (PageBuilder shared between viewer and backend)